Yoast/wordpress-seo
# Removal of canonical and rel next/prev

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Check if the canonical isn't rendered anymore
* On archive pages with multiple pages (requires more post than fits in one page) verify that the rel next/prev isn't rendered anymore
* I also removed the `og:url`, verify that this one isn't rendered anymore

## UI changes

## Documentation

## Quality assurance


Fixes #
